src/components/redux/store.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const store = () => (
+  <div>store</div>
+);
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md#:~:text=Create%20a%20store%2C%20an%20action%20and%20a%20reducer%20that%20will%20connect%20to%20you%20API%20endpont%20to%20get%20the%20random%20greeting.) you should create a proper `store` to hold the greetings fetched from the API.